packages/next/next-server/server/api-utils.ts

```diff
@@ -43,6 +43,19 @@ export async function apiResolver(
     setLazyProp({ req: apiReq }, 'cookies', getCookieParser(req))
     // Parsing query string
     setLazyProp({ req: apiReq, params }, 'query', getQueryParser(req))
+    // Parsing preview data
+    setLazyProp(
+      { req: apiReq },
+      'previewData',
+      getPreviewDataParser(apiReq, apiRes, apiContext)
+    )
+    // Parsing preview
+    setLazyProp(
+      { req: apiReq },
+      'preview',
+      () => !!apiReq.previewData || undefined
```

Comment:
This logic does not seem correct.

Reply:
The logic's a little weird, but it returns true || undefined, keeping consistent with how it appears in getStaticProps